DOL.WHD.Section14c.Test/Helpers/ApplicationDocumentHelperTests.cs

```diff
@@ -17,42 +18,59 @@ public class ApplicationDocumentHelperTests
     {
         private Mock<IApplicationService> _mockApplicationService;
         private Mock<IAttachmentService> _mockAttachmentService;
-
+        private Mock<IResponseService> _mockResponseService;
+        private string testFilePath;
         public ApplicationDocumentHelperTests()
         {
             _mockApplicationService = new Mock<IApplicationService>();
             _mockAttachmentService = new Mock<IAttachmentService>();
+            _mockResponseService = new Mock<IResponseService>();
         }
 
         [TestInitialize]
         public void Initialize()
         {
+            testFilePath = Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"..\..\..\TestFiles"));
+            if (!Directory.Exists(testFilePath))
+                testFilePath = Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"..\..\TestFiles"));
+            EventLog.WriteEntry("Test", testFilePath);
+            EventLog.WriteEntry("Test", AppDomain.CurrentDomain.BaseDirectory);
```

Comment:
Do these still need to be here?  They look like they're here for debugging.  (Nice way of capturing those directories, btw!)  They made the tests fail on my VM, something about the entry not being found or something.  When I removed these two `EventLog` lines, the tests passed with flying colors.

Reply:
Good catch. I forget to remove my debug code.